etc/web/aliases-py26.conf

```diff
@@ -2,7 +2,7 @@
 WSGIScriptAlias /accounts                /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/account.py
 WSGIScriptAlias /accountlimits           /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/account_limit.py
 WSGIScriptAlias /auth                    /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/authentication.py
-WSGIScriptAlias /credentials             /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/credentials.py
+WSGIScriptAlias /credentials             /opt/rucio/.venv/lib/python2.6/site-packages/rucio/web/rest/credential.py
```

Comment:
We need to feature this on the release notes, existing installations need to rename the file. @bari12

Reply:
Maybe I missed something, why was this renamed? 😄

Reply:
The REST file has always been named credential.py as far as I can see, but the paths in the aliases files referred to credentials.py. I corrected them to match the actual filename.